modules/dshow/dshow.cpp

```diff
@@ -43,10 +45,11 @@ IID IID_ISampleGrabberCB = {
 
 #include "qedit.h"
 
+/*
 const CLSID CLSID_SampleGrabber = { 0xc1f400a0, 0x3f08, 0x11d3,
   { 0x9f, 0x0b, 0x00, 0x60, 0x08, 0x03, 0x9e, 0x37 }
 };
-
+*/
```

Comment:
this was added in commit 2076043d899a575c524a9c7132226d4461836aaf

is it safe to delete it ?